test/e2e/e2e_test.go

```diff
@@ -108,68 +133,119 @@ func TestTaskRun(t *testing.T) {
 		}
 	})
 
-	t.Run("TaskRun Cleanup", func(t *testing.T) {
+	t.Run("check deletion", func(t *testing.T) {
 		if err := wait.PollImmediate(1*time.Second, 1*time.Minute, func() (done bool, err error) {
-			tr, err := c.TaskRuns(ns).Get(ctx, tr.GetName(), metav1.GetOptions{})
-			if k8serrors.IsNotFound(err) {
-				return true, nil
+			_, err = tc.TaskRuns(defaultNamespace).Get(ctx, tr.GetName(), metav1.GetOptions{})
+			if err != nil {
+				if k8serrors.IsNotFound(err) {
+					return true, nil
+				}
+				t.Errorf("Error getting PipelineRun: %v", err)
+				return false, err
 			}
-			t.Logf("Get: %+v, %v", tr.GetName(), err)
 			return false, nil
 		}); err != nil {
-			t.Fatalf("error waiting TaskRun to be deleted: %v", err)
+			t.Fatalf("Error waiting for TaskRun deletion: %v", err)
+		}
+	})
+
+	t.Run("check result", func(t *testing.T) {
+		if resName == "" {
```

Comment:
Wouldn't this test always gets skipped? May be I am missing something but afaiu https://github.com/tektoncd/results/pull/418/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R115 will always be empty, until and unless we get the result name and then run the test

Reply:
ok my bad. The tests run sequentially

Reply:
`resName` is taken from previous tests. We can skip or fail the test here. My intention was there's no need to call an API if the input parameters are blank, as this will definitely fail.